src/arrayMethodSort.js

```diff
@@ -4,8 +4,24 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const n = this.length;
+
+    for (let i = 0; i < n - 1; i++) {
+      for (let j = 0; j < n - 1 - i; j++) {
+        if (compareFunction) {
```

Comment:
It's a good practice to provide a default compare function that mimics the behavior of the standard Array.prototype.sort method when no compareFunction is provided. This will help ensure consistency of sorting behavior.